### DISCUSSION TITLE
LeaderboardAPI.SetScore가 동작하지 않습니다.

### DISCUSSION THREAD

**Discussion by Andwhy81kr** [naverz/zepeto-studio-kor#362] [5. Q&A]
제페토 리더보드셋팅을 마치고,
리더보드 ID를 셋팅해서 구현하였습니다.
유니티 에디터 상에서는 정상적으로 작동 되는데,

PlayOnZepeto로 폰에서 플레이해보면 SetScore다음으로 넘어가지 않습니다.

로그켓으로 로그를 남겨봐도 별다른 로그는 남지 않네요.

디바이스에서 테스트 하기위해 별도의 등록과정이 더 필요한건가요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요. 관련 내용을 확인하기 위해 리더보드 스크린샷 및 관련 코드를 올려주시면 
정확한 도움을 드릴 수 있을 것 같습니다.
감사합니다.

**Comment by Andwhy81kr** [comment]
오늘아침에 같은 문제로 보이는 다른게임이 정상작동하는걸 확인하고,
테스트 해보니 정상작동하는걸 확인하였습니다.
